lib/sqlalchemy/util/_collections.py

```diff
@@ -401,15 +401,15 @@ def to_list(x: Any, default: Optional[List[Any]] = None) -> List[Any]:
         return list(x)
 
 
-def has_intersection(set_, iterable):
+def has_intersection(set_: Union[Set[Any], FrozenSet[Any]], iterable: Iterable[Any]) -> bool:
```

Comment:
iirc there is anyset in the typing module.

But I think we could just use container here, since only __contains__ is needed

Reply:
Changed to `Container`, thx for suggestion